dpdd.tex

```diff
@@ -743,14 +780,14 @@ \subsection{Alerts to \DIASources}
 
 \subsubsection{Information Contained in Each Alert}
 
-For each detected \DIASource, LSST will emit an ``Event Alert" within 60 seconds of the end of visit (defined as the end of image readout from the LSST Camera). These alerts will be issued in \VOEvent format\footnote{Or some other format that is broadly accepted and used by the community at the start of LSST commissioning.}, and should be readable by \VOEvent-compliant clients.
+For each detected \DIASource, LSST will emit an ``Event Alert'' within 60 seconds\req{OTT1} of the end of visit (defined as the end of image readout from the LSST Camera). These alerts will be issued in \VOEvent format\footnote{Or some other format that is broadly accepted and used by the community at the start of LSST commissioning.}, and should be readable by \VOEvent-compliant clients.\dmreq{0002}
 
 \vspace{1em}
 Each alert (a \VOEvent packet) will at least include the following:
 
 \begin{itemize}
-\item {\em alertID}: An ID uniquely identifying this alert. It can also be used to execute a query against the \DB as it existed when this alert was issued
-\item {\em \DB ID}   %: For example, {\tt DR5L1}
+\item \emph{alertID}: An ID uniquely identifying this alert. It can also be used to execute a query against the \DB as it existed when this alert was issued.\dmreq{0274}
+\item \emph{\DB ID}   %: For example, \texttt{DR5L1}
 \item Science Data:
     \begin{itemize}
     \item The \DIASource record that triggered the alert
```

Comment:
The last item in this list should be `Matching \Object IDs from the latest Data Release`.  I don't think we want to link L1 DIAObjects with L2 DIAObjects this closely, and Objects are definitely matched to DIASources.